README.md

```diff
@@ -30,8 +30,9 @@
     - [Partition by top-level context](#partition-by-top-level-context)
     - [Using `Set-Cookie` with `Partitioned`](#using-set-cookie-with-partitioned)
     - [Example usage](#example-usage)
-        - [Using Partitioned for third-party embeds](#using-partitioned-for-third-party-embeds)
-        - [CDN load balancing and headless CMS](#cdn-load-balancing-and-headless-cms)
+        - [Third-party locator service](#third-party-locator-service)
```

Comment:
Try to name this section the same as the section in Key Scenarios.

Reply:
Won't it break the links if they have the same exact name? Both links will only point to one section IIUC